src/mixins/boundary.js

```diff
@@ -0,0 +1,60 @@
+import * as THREE from "three";
+
+export const BoundaryMixin = (superclass) => class extends superclass {
+
+    constructor(config) {
+        super(config);
+        this.boundaryConditions = config.boundaryConditions || {};
+    }
+
+    /**
+     * Boundaries are drawn only if type is "bc1", "bc2", or "bc3".
+     */
+    get areBoundariesEnabled() {return ["bc1", "bc2", "bc3"].includes(this.boundaryConditions.type)}
+
+    /**
+     * Returns a plane-like mesh object with given corner vertices in counterclockwise order.
+     */
```

Comment:
1. input params need explanation
2. plane is defined by 3 points, why have 4?

Reply:
This is not a plane, it's plane-like mesh object with 4 corners.

Reply:
Docstring + mention that c axis of the cell and z axis of coordinate system are always aligned by convention

Reply:
Added.